application/account-management/WebApp/package.json

```diff
@@ -6,26 +6,13 @@
     "dev": "npm run update-translations && npm run swagger && rsbuild dev",
     "build": "npm run update-translations && npm run swagger && rsbuild build",
     "update-translations": "lingui extract && lingui compile --typescript",
-    "check": "biome check",
-    "lint": "biome lint --write",
-    "format": "biome format --write",
     "swagger": "openapi-typescript shared/lib/api/AccountManagement.Api.json -o shared/lib/api/api.generated.d.ts",
     "publish": "npm run build && mkdir -p ../Api/publish/WebApp/dist/ && rsync -r ./dist/ ../Api/publish/WebApp/dist/"
   },
-  "dependencies": {
-    "react-aria-components": "^1.2.1",
-    "react-dom-confetti": "^0.2.0"
-  },
   "devDependencies": {
     "@repo/build": "*",
     "@repo/config": "*",
     "@repo/infrastructure": "*",
     "@repo/ui": "*"
-  },
-  "postcss": {
```

Comment:
This is needed to avoid an additional config file

Reply:
I’ve tested this. If I completely remove it from the self-contained systems, the Tailwind fails (as expected). When I add it to the `package.json` in the root, Tailwind for each self-contained system works again. So this will both simplify the `package.json` for each self-contained system and give us operational consistency.

Reply:
unexpected but interesting